libs/output-mapping/src/TableLoader.php

```diff
@@ -309,14 +316,7 @@ private function buildLoadOptions(
         }
 
         if ($treatValuesAsNullConfiguration !== null) {
-            if ($storageSources->getTable()) {
-                $loadOptions['treatValuesAsNull'] = $treatValuesAsNullConfiguration;
-            } else {
-                $this->logger->warning(sprintf(
-                    'Treating values as null for table "%s" was skipped.',
-                    $source->getDestination()->getTableName(),
-                ));
-            }
+            $loadOptions['treatValuesAsNull'] = $treatValuesAsNullConfiguration;
```

Comment:
Posila se vzdy, i pri zalozeni tabulky z CSV. To ted nicemu nevadi, protoze ten endpoint dovoluje extraKeys.